include/mbedtls/arc4.h

```diff
@@ -96,10 +96,9 @@ void mbedtls_arc4_free( mbedtls_arc4_context *ctx );
 /**
  * \brief          Perform an ARC4 key schedule.
  *
- * \param ctx      The ARC4 context to be setup.
- *                 Must not be \c NULL.
+ * \param ctx      The ARC4 context to be setup. Must be initialized.
```

Comment:
As in the other PRs: let's use full sentences, including pronouns.

Reply:
Fixed.